src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/ConnectionCompleteNotification.cs

```diff
@@ -64,6 +64,11 @@ public class ConnectionCompleteParams
         /// Additional optional message with details about why the version isn't supported.
         /// </summary>
         public string? UnsupportedVersionMessage { get; set; }
+
+        /// <summary>
+        /// Process ID for the connection, if the connection was successful.
+        /// </summary>
+        public string? ServerConnectionId { get; set; }
```

Comment:
why do we have 3 different names for the same thing in STS? ServerProcessId (defined in MDS), ServerConnectionId and PID. 

If we defined it as PID in the protocol, I'd use it and get rid of the ServerConnectionId.